test/e2e-go/cli/goal/expect/goalExpectCommon.exp

```diff
@@ -726,7 +742,7 @@ proc ::AlgorandGoal::SignTransaction { WALLET_NAME WALLET_PASSWORD INPUT_TXN OUT
         expect {
             timeout { ::AlgorandGoal::Abort "Timed out signing transaction"  }
 	        "Please enter the password for wallet '$WALLET_NAME':" { send "$WALLET_PASSWORD\r" ; exp_continue}
-	        eof
+	        eof { ::AlgorandGoal::Abort "EOF signing transaction"  }
```

Comment:
`exp_continue` causes `eof` to be called as well so this test most likely will fail